.github/workflows/main-checks.yml

```diff
@@ -3,11 +3,11 @@ name: Main Checks
 on:
   pull_request:
     paths:
-      - ".github/workflows/main-checks.yml"
-      - "ci/**"
-      - "packages/**/*"
-      - "Cargo.toml"
-      - "Cargo.lock"
+      - '.github/workflows/main-checks.yml'
+      - 'ci/**'
+      - 'packages/**/*'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
```

Comment:
Nit: it would be better if you didn't change the formatting in this PR

Reply:
> Nit: it would be better if you didn't change the formatting in this PR

Oh..It's actually the automatic formatting by my VSCode. I'll change it back later.

Reply:
This still needs to be resolved. Lots of unnecessary changes here